data/battle_anim_scripts.s

```diff
@@ -134,7 +134,7 @@ gBattleAnims_Moves::
 	.4byte Move_FOCUS_ENERGY
 	.4byte Move_BIDE
 	.4byte Move_METRONOME
-	.4byte Move_MIRROR_MOVE @ doesn�t have an actual animation
+	.4byte Move_MIRROR_MOVE @ doesn�t have an actual animation
```

Comment:
What's that?

Reply:
thats text encoding for you... resolved in f1c82abc7e01c2e617d00ad3e7f789a8b02de477